docs/advanced/async.rst

```diff
@@ -393,6 +397,42 @@ received immediately, but only during a connection operation, such as a query.
     # got this: Notify(channel='mychan', payload='hey', pid=961823)
     # (1,)
 
+If you need to use the connection to handle :sql:`NOTIFY` in a timely manner
+you can combine the two methods.
```

Comment:
What you describe is your example is not receiving notifications in a timely matter. The notification will be only received when the client is ready to receive it, not when the server sends it.

Reply:
>What you describe is your example is not receiving notifications in a timely matter. The notification will be only received when the client is ready to receive it, not when the server sends it.

That's up for discussion. 

If you need to use the connection for other queries, combining the handler, the generator, and a queue, you will receive notifications as soon as possible. The notifications are stored, and available by First in, first out.
It's the speed (and content) of what the client does based on the notify which will determine whether a client handles the notification in a timely manner.

If someone would receive a Notify every second, and needs more than a second per Notify, he'll never catch up. It doesn't matter what available method the client uses.

But your input does show I need to improve the text. I'll try to clarify. Thanks you for the input.